PaddleCV/image_classification/utils/utility.py

```diff
@@ -468,7 +493,8 @@ def best_strategy_compiled(args, program, loss, exe):
             exec_strategy.num_threads = 1
 
         compiled_program = fluid.CompiledProgram(program).with_data_parallel(
-            loss_name=loss.name,
+            loss_name=loss.name if mode == "train" else loss,
+            share_vars_from=share_prog is mode == "val",
```

Comment:
confused with this line

Reply:
I don't change the parameter list of validate function, 
it passes a loss_name(test_fetch_list[0]) not the loss var(test_fetch_vars[0]).
so add a branch here...